templates/site/ide.html

```diff
@@ -30,6 +30,23 @@
             function nodeFilter([_name, data]) {
                 return data.services && data.services.some(service => service.keywords.includes("jupyterhub"))
             }
+
+            function setNodeLinkJupyter(name, data, node_id_suffix) {
```

Comment:
name this `setNodeLink`

The function name should be generic so that we can use the same function on different pages and not need to have a bunch of if/else statements in the setNodeLinkURL function. This is what we've done for the nodeFilter function (for example).